src/components/Form.js

```diff
@@ -61,8 +61,7 @@ export default class Form extends Component {
 
   onChange = (formData, options={validate: false}) => {
     const liveValidate = this.props.liveValidate || options.validate;
-    const errors = liveValidate ? this.validate(formData) :
-                                  this.state.errors;
+    const errors = liveValidate ? this.validate(formData) : [];
```

Comment:
Well that's a possiblily but when `liveValidate` is not enable, you basically have to resubmit the for to have errors back. I could see a UX scenario where you discard errors field by field as soon as they're fixed, but that's the one of the main purpose of `liveValidate` :)